primitives/src/swap/swap.rs

```diff
@@ -259,15 +254,48 @@ impl<AccountId: Clone, BlockNumber: Clone> Swap<AccountId, BlockNumber> {
   /// * `market_maker_swap` - Market maker (limit order) to test against.
   /// * `market_maker_amount_to_receive` - Expected amount the limit order will receive against `market_maker_amount_to_send`.
   /// * `market_maker_amount_to_send` - Expected amount the market order will receive against `market_maker_amount_to_receive`.
```

Comment:
These need updating to match the new parameter names.

Reply:
Updated.